client/components/suggestion/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
```

Comment:
Nothing in this component is specific to addresses. The server could potantially return a suggestion for any step. For example, if we detect that the merchant chose Media Mail for something that's _suspiciously_ big to be a book (but not over the official limit), we could warn him.